scripts/sentiment_analysis/index.rst

```diff
@@ -48,31 +48,31 @@ epoch:
++----------------+----------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------+
 
 
 test accuracy (SST-1, SST-2, and TREC) or cross-validation accuracy (MR, Subj, CR and MPQA):
 
 +----------------+----------+-----------+-----------+----------+----------+----------+----------+
 |                |   MR     |   SST-1   |   SST-2   |   Subj   |   TREC   |    CR    |   MPQA   |
 +================+==========+===========+===========+==========+==========+==========+==========+
-| rand           |   76.0   |   44.9    |   82.0    |   89.4   |   90.0   |   79.1   |   85.4   |
+| rand           |   75.8   |   44.2    |   82.1    |   89.3   |   90.0   |   79.5   |   85.3   |
 +----------------+----------+-----------+-----------+----------+----------+----------+----------+
-| static         |   79.9   |   46.6    |   86.9    |   92.0   |   91.4   |   82.9   |   89.4   |
+| static         |   79.4   |   48.1    |   87.1    |   91.8   |   91.4   |   83.0   |   89.6   |
 +----------------+----------+-----------+-----------+----------+----------+----------+----------+
-| non-static     |   80.2   |   46.1    |   86.5    |   92.4   |   92.4   |   83.4   |   89.4   |
+| non-static     |   80.0   |   47.0    |   85.6    |   91.9   |   92.4   |   82.9   |   89.2   |
 +----------------+----------+-----------+-----------+----------+----------+----------+----------+
-| multichannel   |   80.2   |   47.9    |   86.0    |   92.9   |   92.8   |   83.6   |   89.8   |
+| multichannel   |   80.0   |   48.1    |   85.8    |   92.1   |   93.8   |   83.3   |   89.6   |
 +----------------+----------+-----------+-----------+----------+----------+----------+----------+
```

Comment:
according to https://github.com/dmlc/web-data/blob/master/gluonnlp/logs/sentiment/TREC_multichannel.log
test acc for TREC-multichannel is 93.2

according to https://github.com/dmlc/web-data/blob/master/gluonnlp/logs/sentiment/TREC_non-static.log
test acc for TREC-non-static is 93.2

Can you double-check all the 28 numbers to make sure they are consistent with the uploaded logs?

Reply:
has been corrected.